pac4cli/wpad.py

```diff
@@ -13,10 +13,18 @@
     txdbus.client.basestring = str
 
 
+_dbusclient = None
+@inlineCallbacks
+def get_dbus_client():
+    global _dbusclient
+    if _dbusclient is None or not _dbusclient.connected:
+        _dbusclient = yield txdbus.client.connect(reactor, 'system')
+    return _dbusclient
```

Comment:
I've never used Twisted before, but it says you should call `returnValue(foo)` instead of `return foo`
https://twistedmatrix.com/documents/13.2.0/api/twisted.internet.defer.inlineCallbacks.html

Reply:
Can you open a ticket to Twisted project, regarding this contradiction in the docs?

Reply:
Sent them a pull request: https://github.com/twisted/twisted/pull/1099